_topics/en/multipath-payments.md

```diff
@@ -58,6 +58,10 @@ primary_sources:
 ## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
 ## true" to bold entry
 optech_mentions:
+  - title: "C-Lightning #3809 adds support to C-Lightning for effective sending of multipath payments"
```

Comment:
nits:

- maybe omit "to C-Lightning ", per the format of other entries

- unsure if "effective" if needed, though it is present in the first sentence of the linked-to news

Reply:
Does the entry need to be at the bottom of the list (in the code, the display is sorted)?

Reply:
Thanks, updated for both

Reply:
@jonatack FYI: entries can be anywhere in the YAML; the display logic sorts by date.